code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -489,3 +489,87 @@
 	var/mob/selected_illusion = illusions[1]
 	X.forceMove(get_turf(selected_illusion.loc))
 	selected_illusion.forceMove(current_turf)
+
+// ***************************************
+// *********** Silence
+// ***************************************
+/datum/action/xeno_action/activable/silence
+	name = "Silence"
+	action_icon_state = "silence"
+	desc = "Impairs the ability of hostile living creatures we can see in a 5x5 area. Targets will be unable to speak and hear for 10 seconds, or 15 seconds if they're your Hunter Mark target."
+	ability_name = "silence"
+	plasma_cost = 50
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_SILENCE,)
+	cooldown_timer = HUNTER_SILENCE_COOLDOWN
+
+/datum/action/xeno_action/activable/silence/can_use_ability(atom/A, silent = FALSE, override_flags)
+	. = ..()
+	if(!.)
+		return
+
+	var/mob/living/carbon/xenomorph/impairer = owner //Type cast this for on_fire
+
+	var/distance = get_dist(impairer, A)
+	if(distance > HUNTER_SILENCE_RANGE)
+		if(!silent)
+			to_chat(impairer, "<span class='xenodanger'>The target location is too far! We must be [distance - HUNTER_SILENCE_RANGE] tiles closer!</spam>")
+		return FALSE
+
+	if(!line_of_sight(impairer, A)) //Need line of sight.
+		if(!silent)
+			to_chat(impairer, span_xenowarning("We require line of sight to the target location!") )
+		return FALSE
+
+	return TRUE
+
+
+
+/datum/action/xeno_action/activable/silence/use_ability(atom/A)
+	var/mob/living/carbon/xenomorph/X = owner
+
+	X.face_atom(A)
+
+	var/victim_count
+	for(var/mob/living/target AS in hearers(HUNTER_SILENCE_AOE, A))
```

Comment:
Refactor this to use `cheap_get_humans_near`

Reply:
Done.

Reply:
No it isn't done. Stop resolving reviews you haven't resolved.

Reply:
I forgot to push the commit before I had left for work. My mistake.